VagrantfileA.rb

```diff
@@ -0,0 +1,6 @@
+Vagrant.configure("2") do |config|
+  config.vm.provision "file", source: "license.json", destination: "/tmp/license.json"
+  config.vm.provision "shell", inline: "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py"
```

Comment:
Is there a reason these can't be added as `provision_commands` in the kitchen file?

Example from the docker based tests.

```
  - name: debian-8
    driver_config:
      image: dliappis/debian-devopsci:8
      privileged: true
      provision_command:
        - apt-get update && apt-get -y install python python-dev python-pip build-essential libyaml-dev python-yaml curl wget
```

Reply:
kitchen-vagrant doesn't support `provision_command:`. closest I could find was the `pre_create_command`